lib/tasks/docs.thor

```diff
@@ -191,27 +191,23 @@ class DocsCLI < Thor
     puts '[S3] Begin syncing.'
     docs.each do |doc|
       puts "[S3] Syncing #{doc.path}..."
-      cmd = "aws s3 sync #{File.join(Docs.store_path, doc.path)} s3://devdocs-assets/#{doc.path} --delete --profile devdocs"
+      cmd = "aws s3 sync #{File.join(Docs.store_path, doc.path)} s3://devdocs-documents/#{doc.path} --delete --profile devdocs"
```

Comment:
The permissions are not yet setup? I'm getting an AccessDenied:

```sh
$ bundle exec thor docs:upload --packaged --dryrun
[S3] Begin syncing.
[S3] Syncing vue~3...
fatal error: An error occurred (AccessDenied) when calling the ListObjectsV2 operation: Access Denied
[S3] Done syncing.
[S3 bundle] Begin uploading.
[S3 bundle] Uploading vue~3.tar.gz...(dryrun) upload: public/docs/vue~3.tar.gz to s3://devdocs-downloads/vue~3.tar.gz
[S3 bundle] Done uploading.
```

Reply:
They weren't, no, and I'm going to review the policies before migrating, but I've attached one that should let you upload.

Reply:
The `--dryrun` command succeeds now:

```sh
$ bundle exec thor docs:upload --packaged --dryrun
[S3] Begin syncing.
[S3] Syncing vue~3...
(dryrun) upload: public/docs/vue~3/api/application-api.html to s3://devdocs-documents/vue~3/api/application-api.html
(dryrun) upload: ...
[S3] Done syncing.
[S3 bundle] Begin uploading.
[S3 bundle] Uploading vue~3.tar.gz...
(dryrun) upload: public/docs/vue~3.tar.gz to s3://devdocs-downloads/vue~3.tar.gz
[S3 bundle] Done uploading.
```